generators/server/templates/src/main/java/package/config/_JacksonConfiguration.java

```diff
@@ -1,27 +1,49 @@
 package <%=packageName%>.config;
 
-import <%=packageName%>.domain.util.*;
-import com.fasterxml.jackson.databind.SerializationFeature;
-import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
-import java.time.*;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateDeserializer;
+import com.fasterxml.jackson.datatype.jsr310.ser.ZonedDateTimeSerializer;
+
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import javax.annotation.PostConstruct;
+
 @Configuration
 public class JacksonConfiguration {
 
+    public static final DateTimeFormatter ISO_FIXED_FORMAT =
+        DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'").withZone(ZoneId.of("Z"));
+
+    public static final DateTimeFormatter ISO_DATE_OPTIONAL_TIME =
+         new DateTimeFormatterBuilder()
+            .append(DateTimeFormatter.ISO_LOCAL_DATE)
+            .optionalStart()
+            .appendLiteral('T')
+            .append(DateTimeFormatter.ISO_TIME)
+            .toFormatter();
+
+    @Autowired
+    private Jackson2ObjectMapperBuilder builder;
+
+    //To be replaced by a Jackson2ObjectMapperBuilderCustomizer in Spring-boot 1.4
+    @PostConstruct
+    public void postConstruct() {
+        this.builder.serializers(new ZonedDateTimeSerializer(ISO_FIXED_FORMAT));
+        //Will not be needed anymore with SB 1.4 (Jackson > 2.7.1)
+        this.builder.deserializerByType(LocalDate.class, new LocalDateDeserializer(ISO_DATE_OPTIONAL_TIME));
+    }
+
     @Bean
-    Jackson2ObjectMapperBuilder jackson2ObjectMapperBuilder() {
-        JavaTimeModule module = new JavaTimeModule();
-        module.addSerializer(OffsetDateTime.class, JSR310DateTimeSerializer.INSTANCE);
-        module.addSerializer(ZonedDateTime.class, JSR310DateTimeSerializer.INSTANCE);
-        module.addSerializer(LocalDateTime.class, JSR310DateTimeSerializer.INSTANCE);
-        module.addSerializer(Instant.class, JSR310DateTimeSerializer.INSTANCE);
-        module.addDeserializer(LocalDate.class, JSR310LocalDateDeserializer.INSTANCE);
-        return new Jackson2ObjectMapperBuilder()
-                .featuresToDisable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
-                .findModulesViaServiceLoader(true)
-                .modulesToInstall(module);
+    public ObjectMapper myJacksonObjectMapper() {
```

Comment:
Can you change the name to `jacksonObjectMapper` or `customJacksonObjectMapper` ?